src/libraries/System.Net.Security/tests/FunctionalTests/System.Net.Security.Tests.csproj

```diff
@@ -12,6 +12,8 @@
   </ItemGroup>
   <ItemGroup Condition="'$(TargetsBrowser)' != 'true'">
     <Compile Include="DummyTcpServer.cs" />
+    <Compile Include="MD4Tests.cs" />
```

Comment:
Should these also be constrained to Android only?

Reply:
done